app/pkg/ai/eval.go

```diff
@@ -0,0 +1,223 @@
+package ai
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+
+	"connectrpc.com/connect"
+	"github.com/go-logr/zapr"
+	"github.com/jlewi/cloud-assistant/app/pkg/config"
+	"github.com/jlewi/cloud-assistant/app/pkg/logs"
+	"github.com/jlewi/cloud-assistant/protos/gen/cassie"
+	"github.com/jlewi/cloud-assistant/protos/gen/cassie/cassieconnect"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	"golang.org/x/net/http2"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/proto"
+)
+
+type Assertor interface {
```

Comment:
the idiomatic go name ends in `er`
```suggestion
type Asserter interface {
```

Reply:
fixed